core/src/main/java9/com/linecorp/armeria/internal/common/Java9VersionSpecific.java

```diff
@@ -28,22 +28,22 @@
 /**
  * Implementation of {@link JavaVersionSpecific} using Java 9 APIs.
  */
-class Java9VersionSpecific extends JavaVersionSpecific {
+final class Java9VersionSpecific extends JavaVersionSpecific {
```

Comment:
IIUC, this class does not have `final` intentionally. From the discussion https://github.com/line/armeria/pull/2069#discussion_r325577213, another `Java1XVersionSpecific` could inherit `Java9VersionSpecific`.
/cc @anuraaga

Reply:
Good point. Removed `final`. 😄